akka-persistence-tck/src/main/scala/akka/persistence/CapabilityFlags.scala

```diff
@@ -52,3 +52,9 @@ trait SnapshotStoreCapabilityFlags extends CapabilityFlags {
   // no flags currently
 }
 //#snapshot-store-flags
+
+trait CurrentEventsByTagCapabilityFlags extends CapabilityFlags {
```

Comment:
Thanks for including me, Patrik. Also, thanks Dennis for taking a crack at
this. I think especially for the JDBC plugin (RDBMS) this should be done.

Vaughn
On Jun 21, 2016 3:06 PM, "Patrik Nordwall" <notifications@github.com> wrote:

> In
> akka-persistence-tck/src/main/scala/akka/persistence/CapabilityFlags.scala
> <https://github.com/akka/akka/pull/20807#discussion_r67863541>:
>
> > @@ -52,3 +52,9 @@ trait SnapshotStoreCapabilityFlags extends CapabilityFlags {
> >    // no flags currently
> >  }
> >  //#snapshot-store-flags
> > +
> > +trait CurrentEventsByTagCapabilityFlags extends CapabilityFlags {
>
> Thanks for looking into that.
>
> I talked to @VaughnVernon <https://github.com/VaughnVernon> about this at
> ScalaDays and he also says that this is an important thing.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/akka/akka/pull/20807/files/1e7268bb91b8975509db868825ac4c1f78b64c41#r67863541>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACg87Js1k7OKBJZqAXgXNp9CfKPz-91kks5qN-HpgaJpZM4I5gQ6>
> .
>